src/kaleidoscope/plugin/OneShot.h

```diff
@@ -68,45 +62,85 @@ class OneShot : public kaleidoscope::Plugin {
   static void disableStickabilityForModifiers();
   static void disableStickabilityForLayers();
 
+  // --------------------------------------------------------------------------
+  // Global test functions
+
+  static bool isActive();
+  static bool isSticky();
+  static bool isPressed();
+
+  // --------------------------------------------------------------------------
+  // Single-key test functions
+
+  static bool isOneShotKey(Key key) {
+    return (key.getRaw() >= kaleidoscope::ranges::OS_FIRST &&
+            key.getRaw() <= kaleidoscope::ranges::OS_LAST);
+  }
+  static bool isActive(Key key);
+  static bool isSticky(Key key);
+  static bool isPressed(Key key);
   static bool isStickable(Key key);
 
+  // --------------------------------------------------------------------------
+  // Utility function for other plugins to cancel OneShot keys
+  static void cancel(bool with_stickies = false);
+
+  // --------------------------------------------------------------------------
+  // Vestigial functions?
+  void inject(Key mapped_key, uint8_t key_state);
   static bool isModifierActive(Key key);
```

Comment:
These two functions don't seem to provide anything useful. Perhaps @algernon can enlighten me?